### ISSUE TITLE
High contrast mode issues

### ISSUE THREAD

**Issue by Acrophost** [ckeditor/ckeditor5#16291]
This is an aggregate issue - any problems with CKEditor 5 + high contrast mode in Windows should be reported under this one :)

**Comment by Acrophost** [comment]
## List bullets/numbers do not have changed color in high contrast mode

### Steps

1. While using Windows turn on the high contrast mode in the OS’s settings (it’s best visible with “Night Sky” preset)
2. Add a list in content

### Result

In Night Sky preset, the list numbering/bullets are not visible


https://github.com/ckeditor/ckeditor5/assets/66953380/ee390f39-91c8-4e21-813c-3a9e694d0fbc

**Comment by Acrophost** [comment]
## Cannot tell the state of reversed order switch in Night Sky preset of high contrast mode

### Steps

1. While using Windows turn on the high contrast mode in the OS’s settings (choose the “Night Sky” preset)
2. Make a numbered list in the content
3. Open numbered list styles and list properties section

### Result

The reverse order switch is a black void

![Screenshot_1](https://github.com/ckeditor/ckeditor5/assets/66953380/d224bd3c-59c3-4b23-96b9-023ecd306c2b)
